README.md

```diff
@@ -27,7 +27,11 @@ You can retrieve binaries for Linux, macOS and Windows on our [GitHub release pa
 ##### Linux
 If you prefer to do it straight from CLI here's a one-liner for you:
 ```bash
-curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
+curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJ ./opencompose
+```
+and if you want it installed system-wide:
+```bash
+sudo mv ./opencompose /usr/local/bin/
```

Comment:
Not to sure how to give an example, just following usual conventions when copying files over. I've done the mistake personally of auto-tabbing a file when supply `/` and overriding something by accident.